src/transaction/transaction.proto

```diff
@@ -22,8 +22,10 @@ message TransactionRequest {
   pfs_v2.CreateBranchRequest create_branch = 6;
   pfs_v2.DeleteBranchRequest delete_branch = 7;
   pps_v2.UpdateJobStateRequest update_job_state = 8;
-  pps_v2.CreatePipelineRequest create_pipeline = 9;
+  reserved 9;
+  reserved "create_pipeline";
```

Comment:
Do we have any formal deprecation process for our APIs? I don't have much issue with this particular change, but it happened recently with the debug dumpv2 API as well so I think it's worth bringing up.

Reply:
> Do we have any formal deprecation process for our APIs?

I think the formal process is getting team sign-offs on the proto changes that we started in the last 48 hours 😁 

In this particular case, the issue is that the transaction request message formerly just contained a create pipeline request, whereas now it needs to contain three items of data. Rather than encode that knowledge of PPS information within the transaction service, I think it makes sense for it to pull in a create pipeline transaction message from PPS itself.